README.md

```diff
@@ -392,6 +392,8 @@ are not maintained by the `tokio` project. These include:
   _inside_ of functions.
 - [`tracing-wasm`] provides a `Collector`/`Subscriber` implementation that reports
   events and spans via browser `console.log` and [User Timing API (`window.performance`)].
+- [`tracing-web`] provides a layer implementation of level-aware logging of events to web browsers' `console.*` and span events to
+  the [User Timing API (`window.performance`)].
```

Comment:
let's wrap this line at the same column as the other lines in the file?

Reply:
I hope I got the line correctly, I didn't find any guidance on the exact number. Is there a rustfmt.toml file somewhere that specifies this?